client/my-sites/plan-features/style.scss

```diff
@@ -91,6 +91,26 @@ $plan-features-sidebar-width: 272px;
 	}
 }
 
+.plan-features__table-row {
+	&.is-highlighted {
```

Comment:
In this case, should just write it as `.plan-features__table-row.is-highlighted {}`